Feature/#170 documentation for working with tags versioned

BatJan opened the pull request:
Adding a YAML "tagged" version of the current tags documentation + legacy documentation for versions before Umbraco 7.6 using the vpost file prefix.

This PR relates to #170 and #900

dampee commented:
Hi Jan
Because we merged the other pull request, this one is now having conflicts.
Can you rebase (or cherry pick the right commits)?

Really looking forward on merging this.  But it has to wait for umbraco/OurUmbraco#236 to be merged.  Right?

BatJan commented:
I'll have a look and make sure to sort out the troubles later today :)

BatJan commented:
Hi @dampee I think this is good for another review? :)

dampee commented on Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags-vpost-7.0.md:
```diff
@@ -0,0 +1,77 @@
+---
+keywords: tags property editors v7 version7
+versionFrom: 7.0.0
```
How sure are you about the versionFrom?

BatJan replied:
Gut feeling to be honest - I can try and setup some 7 and 7.3 to see if there are any differences between the version though. I think that should be sufficient, no? Or we can wait and see if anyone reports issues in the forum if/when they come across it?

BatJan replied:
Yup, agree. Just forgot how much has really happened since v7.0 was fresh and shiny! :) - It's been updated now.

dampee commented:
Looks ace indeed!

BatJan commented:
Wee, first mover! 💃

BatJan commented:
Hi @dampee - Just did some dogfooding and noticed that image refernces were broken and the @ sign was missing in the code examples when using the @foreach loop.

Also learned that the documentation I have done pre 7.6 only work from 7.4 where models builder was introduced so I have renamed the filename and updated the yaml tags.
